test/avr/.vscode/launch.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
we already have a "tests" folder, can you move all the ``/test/avr/`` files under there?